spec/views/hq/sponsors/_filters_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe "hq/sponsors/filters.html.haml", type: :view do
+  let(:sponsors_filters) {{
+    "name" => {
```

Comment:
That is what I wanted too but since we don't have a Model for the filters, `FactoryGirl` didn't let me create a factory for it.
Any tips are welcome.